superset/databases/api.py

```diff
@@ -1524,11 +1539,61 @@ def import_(self) -> Response:
         command.run()
         return self.response(200, message="OK")
 
+    @expose("/csv_upload_metadata/", methods=("POST",))
```

Comment:
@dpgaspar would it make this more restful perhaps to say `/csv_metadata` as the endpoint name, and in the process of creating (POST) csv_metadata, the upload action/portion is just an implementation detail?